scripts/exregional_run_fcst.sh

```diff
@@ -449,18 +455,96 @@ cat > itag <<EOF
 /
 EOF
 fi
+#
+#-----------------------------------------------------------------------
+#
+# Update or copy the FV3 input.nml file
+#
+#-----------------------------------------------------------------------
+#
+if [ "${DO_ENSEMBLE}" = TRUE ] && ([ "${DO_SPP}" = TRUE ] || [ "${DO_SPPT}" = TRUE ] || [ "${DO_SHUM}" = TRUE ] || \
+   [ "${DO_SKEB}" = TRUE ] || [ "${DO_LSM_SPP}" =  TRUE ]); then
+  python3 $USHdir/set_FV3nml_ens_stoch_seeds.py \
+      --path-to-defns ${GLOBAL_VAR_DEFNS_FP} \
+      --cdate "$CDATE" || print_err_msg_exit "\
+Call to function to create the ensemble-based namelist for the current
+cycle's (cdate) run directory (DATA) failed:
+  cdate = \"${CDATE}\"
+  DATA = \"${DATA}\""
+else
+  cp_vrfy "${FV3_NML_FP}" "${DATA}/${FV3_NML_FN}"
+fi
+#
+#-----------------------------------------------------------------------
+#
+# Replace parameter values for restart in FV3 input.nml and model_configure.
+# Add restart files to INPUT directory.
+#
+#-----------------------------------------------------------------------
+#
+flag_fcst_restart="FALSE"
+if [ "${DO_FCST_RESTART}" = "TRUE" ] && [ "$(ls -A ${DATA}/RESTART )" ]; then
+  cp_vrfy input.nml input.nml_org
+  cp_vrfy model_configure model_configure_org
```

Comment:
`org` is unclear here. How about `orig`?

Reply:
Updated.